### PR TITLE
Added sample to demonstrate automated screenshot testing with Showkase and Paparazzi

### PR DESCRIPTION
**Pull request by vinaygaba**
Added sample to demonstrate automated screenshot testing with Showkase and Paparazzi. This leverages the example that @alexvanyo created for the Now In Android app https://github.com/android/nowinandroid. 

This is purely meant for demo purposes and will be referenced in the documentation that I'm about to add shortly.

### CODE REVIEW

File: showkase-screenshot-testing-paparazzi-sample/src/main/java/com/airbnb/android/showkase/screenshot/testing/paparazzi/sample/Scrollable.kt [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.airbnb.android.showkase.annotation.ShowkaseComposable
+
+@Composable
+fun VerticalScrollSample() {
+    Column(
```
Comment (oas004) [inline showkase-screenshot-testing-paparazzi-sample/src/main/java/com/airbnb/android/showkase/screenshot/testing/paparazzi/sample/Scrollable.kt:19]: Is there a reason why we are not using LazyColumn here? 😄

**Review by oas004 — APPROVED** [review]
I think this looks very nice! Amazing work! 😄 🤩

**Comment by mochadwi** [comment]
related: https://github.com/android/nowinandroid/pull/101

File: showkase-screenshot-testing-paparazzi-sample/src/main/java/com/airbnb/android/showkase/screenshot/testing/paparazzi/sample/Scrollable.kt (hunk 1 above)
Comment (vinaygaba) [reply showkase-screenshot-testing-paparazzi-sample/src/main/java/com/airbnb/android/showkase/screenshot/testing/paparazzi/sample/Scrollable.kt:19]: no, was just added as a sample. No particular reason to add it or skip it 😅

File: showkase-screenshot-testing-paparazzi-sample/src/test/java/com/airbnb/android/showkase/screenshot/testing/paparazzi/sample/PaparazziSampleScreenshotTest.kt [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+import com.airbnb.android.showkase.models.ShowkaseBrowserColor
+import com.airbnb.android.showkase.models.ShowkaseBrowserTypography
+import com.airbnb.android.showkase.ui.padding4x
+import java.util.*
+
+/**
+ * Credit to Alex Vanyo for creating this sample in the Now In Android app by Google.
+ * PR here - https://github.com/android/nowinandroid/pull/101. Modified the test from that PR to
+ * my own needs for this sample.
+ */
```
Comment (pmecho) [inline showkase-screenshot-testing-paparazzi-sample/src/test/java/com/airbnb/android/showkase/screenshot/testing/paparazzi/sample/PaparazziSampleScreenshotTest.kt:39]: Do we need to include the apache license at the top?

**Review by pmecho — APPROVED** [review]

**Comment by anhanh11001** [comment]
How can you get `getMetadata()` extension function here? I follow similar approach and also compile my app but never receive this function

**Comment by vinaygaba** [comment]
@anhanh11001 Sources generated from ksp would need to be registered separately, similar to how it's done in the sample app of the project - https://github.com/airbnb/Showkase/pull/279

**Pull request merged**